hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/ResourceAttributeEnricher.java

```diff
@@ -63,6 +71,18 @@ public void enrichEvent(StructuredTrace trace, Event event) {
                       }
                       return attributeValue;
                     }));
+
+        // Add deployment attribute if pod name is available
+        if (resourceAttributeKey.equals(POD_NAME_KEY)) {
```

Comment:
Add this if condition in the above code block. After the node_selector_key if condition.

Reply:
not sure how it will be possible. In the above code block, we are modifying the attributeValue if resourceAttributeKey is not present in the map, but I want to add the same if a specific key (deployment) is not present.